deployment/fedora-package-x64/pkg-src/jellyfin.spec

```diff
@@ -12,7 +12,7 @@ Release:        1%{?dist}
 Summary:        The Free Software Media Browser
 License:        GPLv2
 URL:            https://jellyfin.media
-Source0:        %{name}-%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.xz
```

Comment:
This would break, your script can create both `.tar.gz` and `.tar.xz`. I suggest to stick to `.tar.gz` as a universal variant, not everybody has `xz` installed.

Reply:
Fixed